README.md

```diff
@@ -38,6 +38,16 @@ Which brings us to...
 
 Yup, that's it. You can proxy requests to the Go service through Apache/nginx/etc., but that's not required.
 
+## Docker
+* docker 1.4+
+
+You should follow the quickstart guide up to step (2) and then run:
```

Comment:
Seems like this should go through step 3, right? That way config.json will exist.